src/state/slices/portfolioSlice/portfolioSliceCommon.ts

```diff
@@ -8,9 +8,9 @@ export const supportedAccountTypes = {
     UtxoAccountType.SegwitP2sh,
     UtxoAccountType.P2pkh,
   ],
-  [ChainTypes.Ethereum]: undefined,
-  [ChainTypes.Cosmos]: undefined,
-  [ChainTypes.Osmosis]: undefined,
+  [ChainTypes.Ethereum]: [undefined],
+  [ChainTypes.Cosmos]: [undefined],
+  [ChainTypes.Osmosis]: [undefined],
```

Comment:
why not just an empty array?
```suggestion
  [ChainTypes.Ethereum]: [],
  [ChainTypes.Cosmos]: [],
  [ChainTypes.Osmosis]: [],
```

Reply:
https://github.com/shapeshift/web/pull/1520/files#diff-c2af5485a28352209ae9b9f3640dbe44d6ba42e73215bc109bc7419ca1af95daR98

we need to map over a non zero length array to avoid chain specific logic

Reply:
added comment to explain.